src/webserver/BaseWebServer.js

```diff
@@ -7,22 +7,15 @@ module.exports = function (
   ErrorMiddleware,
   config,
   ControllerRegistration,
-  WinstonRequestLoggingMiddleware
+  WinstonRequestLoggingMiddleware,
 ) {
   class BaseWebServer {
-
     constructor() {
       this.app = express();
     }
 
     getPort() {
-      let port = config.Port;
-
-      if (this.server) {
-        port = this.server.address().port || port;
-      }
-
-      return port;
```

Comment:
Simplified logic because `port` property is always defined for `this.server.address()`